src/azure-cli-core/azure/cli/core/_azure_env.py

```diff
@@ -16,22 +16,28 @@ class ENDPOINT_URLS: #pylint: disable=too-few-public-methods,old-style-class,no-
     MANAGEMENT = 'management'
     ACTIVE_DIRECTORY_AUTHORITY = 'active_directory_authority'
     ACTIVE_DIRECTORY_GRAPH_RESOURCE_ID = 'active_directory_graph_resource_id'
+    KEY_VAULT = 'key_vault'
 
 _environments = {
     ENV_DEFAULT: {
         ENDPOINT_URLS.MANAGEMENT: 'https://management.core.windows.net/',
         ENDPOINT_URLS.ACTIVE_DIRECTORY_AUTHORITY : 'https://login.microsoftonline.com',
-        ENDPOINT_URLS.ACTIVE_DIRECTORY_GRAPH_RESOURCE_ID: 'https://graph.windows.net/'
+        ENDPOINT_URLS.ACTIVE_DIRECTORY_GRAPH_RESOURCE_ID: 'https://graph.windows.net/',
+        ENDPOINT_URLS.KEY_VAULT: 'https://vault.azure.net'
         },
     ENV_CHINA: {
         ENDPOINT_URLS.MANAGEMENT: 'https://management.core.chinacloudapi.cn/',
         ENDPOINT_URLS.ACTIVE_DIRECTORY_AUTHORITY: 'https://login.chinacloudapi.cn',
-        ENDPOINT_URLS.ACTIVE_DIRECTORY_GRAPH_RESOURCE_ID: 'https://graph.chinacloudapi.cn/'
+        ENDPOINT_URLS.ACTIVE_DIRECTORY_GRAPH_RESOURCE_ID: 'https://graph.chinacloudapi.cn/',
+        # TODO: Verify KeyVault endpoint
+        ENDPOINT_URLS.KEY_VAULT: 'https://vault.chinacloudapi.cn'
```

Comment:
Just cross check from xplat [environment.js](https://github.com/Azure/azure-xplat-cli/blob/dev/lib/util/profile/environment.js#L181) where keyvault team put in the endpoints themselves. If still missing, please ask @karlaugsten